<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addlesson</class>
 <widget class="QDialog" name="addlesson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>619</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QLabel" name="picLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>420</width>
     <height>600</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>فصل</string>
   </property>
   <widget class="QComboBox" name="comboBoxCat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOther">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>570</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>افزودن</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>افزودن در</string>
   </property>
   <widget class="QComboBox" name="comboBoxGradeCat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>پنجم</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ششم</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>هفتم</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>هشتم</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>اول متوسطه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>دوم متوسطه</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>آمار</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>سوم (ریاضی)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>سوم (تجربی)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>چهارم (ریایضی)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>چهارم (تجربی)</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonBrowse">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>انتخاب عکس</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>540</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>بستن این پنجره بعد از افزودن</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>picLabel</zorder>
  <zorder>pushButtonAdd</zorder>
  <zorder>pushButtonBrowse</zorder>
  <zorder>checkBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
